modules/gitops-release-notes-1-4-0.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-0_{context}"]
+= Release notes for {gitops-title} 1.4.0
+
+{gitops-title} 1.4.0 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="new-features-1-4-0_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.4.0:
```

Comment:
IBM SG 22, avoid self referential language, like "this section describes/highlights"

If this phrasing is standard for `New features` keep it. You want to be consistent with the rest of OpenShift's `New features` content. 

If you can change it, you can rephrase to something like, 
`In addition to seeing the fixes and stability improvements, see the following for what is new in Red Hat OpenShift GitOps 1.4.0:`

Reply:
Updated. Thanks.